core/include/detray/propagator/navigator.hpp

```diff
@@ -410,8 +414,9 @@ class navigator {
     DETRAY_HOST_DEVICE inline bool init(propagator_state_t &propagation) const {
 
         state &navigation = propagation._navigation;
+        const auto &det = navigation.detector();
```

Comment:
Does this return a reference to a pointer? I am wondering, does simply using ```const auto``` make things easier or does that break something?

Reply:
You are right :p That was dumb.